include/common/roundeven.h

```diff
@@ -56,7 +56,7 @@ inline float roundevenf(float Value) {
   return Ret;
 #elif defined(__aarch64__)
   float Ret;
-  __asm__("frintn %s0, %s0" : "=w"(Ret) : "w"(Value));
+  __asm__("frintn %s0, %s1" : "=w"(Ret) : "w"(Value));
```

Comment:
Is this failed only on the macOS m1, or on any aarch64 platform?
If it's only related to macOS, then you must identify that in a `defined` condition.

Reply:
Ah, should apply to all aarch64. Thanks.